src/plugins/intel_gna/tests/unit/gna_memory_alignment.cpp

```diff
@@ -60,33 +52,41 @@ class GNAPluginForMemoryAlignmentTest : public GNAPlugin {
         }
     }
 
-    size_t get_memory_REGION_INPUTS_size() {
+    const size_t get_memory_REGION_INPUTS_size() const {
```

Comment:
we don't need to mark return value as const

Reply:
@tadamowicz , could you please take this comment into account in your next PR?